Makefile

```
@@ -0,0 +1,19 @@
+.PHONY: up down build bash install test
+
+up:
+	docker-compose up -d
+
+down:
+	docker-compose down
+
+build:
+	docker-compose up -d --build
+
+bash:
+	docker-compose exec php bash
+
+install:
+	docker-compose exec php composer install
+
+test:
+	docker-compose exec php ./vendor/bin/phpunit
```

```suggestion
	docker-compose exec php ./vendor/bin/phpunit
	
	test-filter:
	docker-compose exec php ./vendor/bin/phpunit --filter=$(filter)
```
Minor nitpick, but there are >100 tests. so might be handy if you want to debug only 1.